pkg/topology/full/full_test.go

```diff
@@ -128,7 +140,8 @@ func TestAddPeer(t *testing.T) {
 
 	t.Run("OK - connected peers - peer not already connected", func(t *testing.T) {
 		discovery := mock.NewDiscovery()
-		addressbook := inmem.New()
+		inmem := mockstate.NewStateStore()
```

Comment:
```suggestion
		statestore:= mockstate.NewStateStore()
```

all the above where `inmem` name is used.